cmd/tdex/config.go

```diff
@@ -197,11 +197,11 @@ func configConnectAction(c *cli.Context) (err error) {
 
 	if err = setState(map[string]string{
 		"proto":          proto,
-		"no_tls":         strconv.FormatBool(proto == "http"),
+		noTlsKey:         strconv.FormatBool(proto == "http"),
 		"rpcserver":      rpcServerAddr,
-		"no_macaroons":   noMacaroons,
-		"tls_cert_path":  tlsCertPath,
-		"macaroons_path": macaroonsPath,
+		noMacaroonsKey:   noMacaroons,
+		tlsCertPathKey:   tlsCertPath,
+		macaroonsPathKey: macaroonsPath,
```

Comment:
revert these changes please, only the flags must change, not the content of the json config file